src/main/metabase/examples.ts

```diff
@@ -132,35 +220,36 @@ function makeExampleInfos(field: Field, table: Table, aidColumns: string[]): Exa
 }
 
 export function exampleQueries(table: Table, aidColumns: string[]): ExamplesSection[] {
-  const exampleInfos = table.fields.flatMap((field) => makeExampleInfos(field, table, aidColumns));
+  const exampleQueries = table.fields.flatMap((field) => columnExampleQueries(field, table, aidColumns));
+
+  function findField(name: string) {
+    const field = find(table.fields, { name });
+    if (!field) throw new Error(`Field '${name}' not found in table ${table.name}.`);
+    return field;
+  }
   // const t = getT('example-queries'); // Let's worry about i18n later...
 
   return [
     {
-      title: 'Overview',
+      title: '# Overview',
       queries: [
-        {
-          name: `Count of ${table.display_name}`,
-          sql: lines('SELECT count(*)', `FROM ${table.name}`),
-          sizeX: 6, // 6 is a good default (3 cards per row).
-          sizeY: 4, // 4 is a good default.
-          display: 'scalar',
-          visualizationSettings: {}, // No visualizations for now. To be done after we finish SQL.
-        },
+        makeQuery(
+          scalar({
+            name: 'Rows in table',
+            sql: lines('SELECT count(*)', `FROM ${table.name}`),
+          }),
+        ),
+        ...aidColumns.map((aidColumn) =>
+          makeQuery({
+            ...countDistinctSQL(findField(aidColumn), table),
+            name: 'Distinct entities',
+          }),
+        ),
       ],
     },
     {
-      title: `Overview of ${table.display_name} columns`,
-      queries: exampleInfos.map(({ name, sql }) => {
-        return {
-          name: name,
-          sql: sql,
-          sizeX: 6,
-          sizeY: 4, // TODO: For a table we might need something taller.
-          display: 'table' as Display, // For now we show results only as 'table'.
-          visualizationSettings: {},
-        };
-      }),
+      title: `# Overview of ${table.display_name} columns`,
+      queries: exampleQueries.map(makeQuery),
```

Comment:
Rename `makeQuery` to `makeCardQuery`, or perhaps rename `exampleQueries` to `exampleInfos`

Reply:
The word "Info" vs "Query" is creating confusion. Not sure what naming convention to use...
Maybe we should make everything optional in `ExampleQuery` and provide defaults when materializing the card.

Reply:
I got rid of `ExampleInfo` entirely. We keep things `undefined` until the last moment. Let me know what you think.

Reply:
agreed. I added "info" for lack of better word, to mean something intermediate, from which the API request structure is built. The idea with defaults could work too, let's do whatever is simpler